Quantum-Control-Applications/NV-Centers/Confocal-Setup/manual_mixer_calibration.py

```diff
@@ -0,0 +1,31 @@
+"""
+Calibration for mixer imperfections
+"""
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm.qua import *
+from configuration import *
+
+with program() as cw_output:
+    with infinite_loop_():
+        play("cw", "NV")
+
+qmm = QuantumMachinesManager(qop_ip)
+qm = qmm.open_qm(config)
+
+job = qm.execute(cw_output)
+
+# When done, the halt command can be called and the offsets can be written directly into the config file.
+
+# job.halt
+
+# These are the 2 commands used to correct for mixer imperfections. The first is used to set the DC of the `I` and `Q`
+# channels to compensate for the LO leakage. The 2nd command is used to correct for the phase and amplitude mismatches
```

Comment:
I would add something like this since several customers seemed confused: 
"Since this compensation depends on the I & Q powers, it is advised to run this step with no input power, so that there is no LO leakage while the pulses are not played."

Reply:
good point